virtool/blast/task.py

```diff
@@ -64,34 +64,56 @@ async def wait(self):
         the search completes. The task completes when either an error or
 
         """
-        try:
-            while True:
-                await asyncio.sleep(self.interval)
 
-                blast = await self.data.blast.check_nuvs_blast(
-                    self.analysis_id, self.sequence_index
-                )
+        for _ in range(0, retries):
```

Comment:
Not sure if a `for` loop makes sense for this. We only want to run the blast analysis once, but will run it more times if we have to.